scenes/world_map/teleporter.gd

```diff
@@ -1,17 +1,72 @@
+@tool
 class_name Teleporter
 extends Area2D
 
-@export_file("*.tscn") var scene_to_go_to: String
+const SPAWN_POINT_GROUP_NAME: String = "spawn_point"
+
+@export_file("*.tscn") var scene_to_go_to: String:
+	set(new_value):
+		scene_to_go_to = new_value
+		_update_available_spawn_points()
+		notify_property_list_changed()
+
+var spawn_point_path: NodePath:
+	set(new_val):
+		if new_val == ^"NONE":
```

Comment:
I see below that `^"NONE"` is something you are building with the hint string. Is it because there is no way to pass the empty path `^""` ?

Reply:
Yes! Since the hint_string expects a comma separated list of values, if you have an empty string, it gets ignored. Or at least it did that when I was testing it. Maybe I did something wrong.

In any case, I also believe that having a NONE element is clearer than having an "" element.